generics/services/entity-management.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * name : entity-management.js
+ * author : Praveen
+ * Date : 13-Jun-2024
+ * Description : Entity service related information.
+ */
+
+//dependencies
+const request = require('request');
+
+const entityManagementServiceUrl = process.env.ENTITY_MANAGEMENT_SERVICE_URL;
```

Comment:
@praveenKDass  define new env variables in env samples and envVariables file

Reply:
Added @VISHNUDAS-tunerlabs